web3.js/src/__forks__/browser/fetch-impl.ts

```diff
@@ -0,0 +1 @@
+export default globalThis.fetch;
```

Comment:
`globalThis` will resolve to `self` in workers, and `window` in browsers. Both have `fetch` available on them.